app/views_test.py

```diff
@@ -725,7 +725,7 @@ def test_valid_get_all_reservations(monkeypatch):
     with app.app_context():
         with app.test_request_context():
             def reservations_found():
-                room = Room(1, False)
+                room = Room(1)
                 user = User('buddy', 'boy')
                 time = Timeslot(1, 2, '2020-01-01', '', 1, 1)
```

Comment:
For example this Timeslot is invalid, since you know have strings for IDs, so the last parameter is invalid..

The integer 1 cannot be a UUID...